install_config/persistent_storage/persistent_storage_gce.adoc

```diff
@@ -142,3 +142,35 @@ formatted with the given file system.
 
 This allows using unformatted GCE volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
+=== Multizone Configuration
+
+In multi-zone configurations, You must specify
```

Comment:
This bit of document fixes somewhat unrelated issue - https://bugzilla.redhat.com/show_bug.cgi?id=1371872 . 

That bug was opened because - users were unable to create PVs (manually) in zones other than zone in which master node is. This bug still exists. To get around this bug, you must specify `failure-domain` as documented here.  

So this documentation is not applicable to PVs created via dynamic provisioning but via static provisioning.